<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextEdit" name="server_console"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="stat">
       <property name="text">
        <string>Statistiques</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_screen">
       <property name="text">
        <string>Effacer la console</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="list_users">
       <property name="text">
        <string>Utilisateurs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Lancer le jeu</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="quit_server">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="launch_server">
       <property name="text">
        <string>Lancer le serveur</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Server</sender>
   <signal>destroyed()</signal>
   <receiver>quit_server</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
